packages/amplify-e2e-core/src/init/amplifyPull.ts

```diff
@@ -1,15 +1,41 @@
 import { getCLIPath, nspawn as spawn } from '../../src';
 
-export function amplifyPull(cwd: string, settings: any) {
+export function amplifyPull(cwd: string, settings: { override?: boolean; emptyDir?: boolean; appId?: string }) {
   return new Promise((resolve, reject) => {
     const tableHeaderRegex = /\|\sCategory\s+\|\sResource\sname\s+\|\sOperation\s+\|\sProvider\splugin\s+\|/;
     const tableSeperator = /\|(\s-+\s\|){4}/;
 
-    const chain = spawn(getCLIPath(), ['pull'], { cwd, stripColors: true })
-      .wait('Pre-pull status')
-      .wait('Current Environment')
-      .wait(tableHeaderRegex)
-      .wait(tableSeperator);
+    const chain = spawn(getCLIPath(), settings.appId ? ['pull', '--appId', settings.appId] : ['pull'], { cwd, stripColors: true });
```

Comment:
are these changes related to LL?

Reply:
It's a generic upgrade to the amplifyPull e2e helper I wrote previously, but it's now also used in a LL e2e test